btcutil/address.go

```diff
@@ -469,9 +463,6 @@ func (a *AddressPubKey) serialize() []byte {
 
 	case PKFCompressed:
 		return a.pubKey.SerializeCompressed()
-
-	case PKFHybrid:
-		return a.pubKey.SerializeHybrid()
```

Comment:
Ended up removing support for hybrid keys, as they don't exist in BIP 340 and have been non standard for some time elsewhere.